src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/MergePrisonerService.kt

```diff
@@ -8,11 +8,18 @@ import org.springframework.stereotype.Service
 
 const val MERGE_EVENT_NAME = "hdc-api.prisoner.merged"
 
+fun interface Done {
+  fun complete()
```

Comment:
This provides something to wait for in the tests.. can't really write a test otherwise for something that isn't going to happen..

so we inject a spy of this into the service and then wait for this to be called. 
As we know this will be the last thing thats called when processing a message, we can safely verify that no telemetry event was raised